vk_video_decoder/libs/VkVideoParser/VulkanVideoParser.cpp

```diff
@@ -2156,18 +2156,18 @@ bool VulkanVideoParser::DecodePicture(
 
         // hevc->irapPicFlag = m_slh.nal_unit_type >= NUT_BLA_W_LP &&
         // m_slh.nal_unit_type <= NUT_CRA_NUT;
-        pStdPictureInfo->flags.IrapPicFlag = pin->IrapPicFlag; // Intra Random Access Point for current picture.
+        pStdPictureInfo->flags.IrapPicFlag = pin->IrapPicFlag & 0x1; // Intra Random Access Point for current picture.
```

Comment:
Please revert these changes and fix VkParserHevcPictureData as follows:

    uint8_t IrapPicFlag;
    uint8_t IdrPicFlag;
    uint8_t short_term_ref_pic_set_sps_flag;
    
    to 
    
    uint32_t IrapPicFlag : 1;
    uint32_t IdrPicFlag   : 1;
    uint32_t short_term_ref_pic_set_sps_flag : 1;